Client/Frontend/Browser/HomePanel/NTPDownloader.swift

```diff
@@ -0,0 +1,378 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+
+private let logger = Logger.browserLogger
+
+struct NTPFocalPoint: Codable {
```

Comment:
Some of these structures will be duplicated with Joel's PR

https://github.com/brave/brave-ios/pull/2203/files#diff-516529ec2ce543f7446b5454f973fe52R11

We can consolidate them after both PR are merged, shouldn't be a problem for now

Reply:
Yeah, let's leave this as-is, and I'll edit when merging. We'll just have to see where makes the most sense. I figured this would probably happen anyway, since we both need the data.